prov/efa/src/rxr/rxr_ep.c

```diff
@@ -367,6 +379,17 @@ void rxr_tx_entry_init(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry,
 	else
 		memset(tx_entry->desc, 0, sizeof(tx_entry->desc));
 
+	/*
+	 * The prefix is currently not used by the sender, but needs to be
+	 * accounted for when copying the payload into the bounce-buffer.
+	 */
+	if (ep->use_zcpy_rx) {
+		assert(tx_entry->iov[0].iov_len >= sizeof(struct rxr_pkt_entry) + sizeof(struct rxr_eager_msgrtm_hdr));
+		tx_entry->iov[0].iov_base = (char *)tx_entry->iov[0].iov_base
+					     + sizeof(struct rxr_pkt_entry)
+					     + sizeof(struct rxr_eager_msgrtm_hdr);
```

Comment:
Isn't `rxr_eager_msgrtm_hdr` supposed to go over the wire?

Reply:
It does, this will just offset the address from which we start copying the user payload into the bounce buffer. pkt_entry->pkt will still point to the base header.

Reply:
Oh, that makes sense. Thanks for the clarification.